bin/atam-codemod-cli.js

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env node
+'use strict';
+
+require('codemod-cli').runTransform(
+  __dirname,
+  process.argv[2] /* transform name */,
```

Comment:
We can hardcode the transform name if it is the only transform we are going to have, do you see any other use case for another codemod?

Reply:
If we have a use case to re-export utils and constants then we can create a new codmod to update the import path in the host app as well as the addon.